src/main/java/calculator/StringCalculator.java

```diff
@@ -0,0 +1,64 @@
+package calculator;
+
+import java.util.Arrays;
+import java.util.Objects;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+public class StringCalculator {
+
+
+    public StringCalculator() {
+    }
+
+    public int add(String text) {
+
+        if (Objects.isNull(text) || text.isBlank()) {
+            return 0;
+        }
+        String[] tokens = getValue(text);
+        String[] numbers = Arrays.stream(tokens)
+                .filter(this::isNumeric)
+                .filter(this::isPositiveNumber)
+                .toArray(String[]::new);
```

Comment:
사소한 의견이지만, 배열보다는 List를 사용하는게 조금더 좋을 것 같아요. 🙂

```suggestion
        List<String> numbers = Arrays.stream(tokens)
            .filter(this::isNumeric)
            .filter(this::isPositiveNumber)
            .collect(Collectors.toList());
```